webpack/assets/javascripts/react_app/common/__mocks__/I18n.js

```diff
@@ -0,0 +1,26 @@
+const cheveronPrefix = () => (window.I18N_MARK ? '\u00BB' : '');
```

Comment:
why is this needed?

Reply:
No needed, another PR created for the I18n:
https://github.com/theforeman/foreman/pull/6162